### PR TITLE
CBO-574: Disbursement page review

### PR DESCRIPTION
**Pull request by cleargif**
#### What
Replacing the `typeahead` with the `accessible autocomplete` component

#### Ticket

https://dsdmoj.atlassian.net/browse/CBO-574

#### Why
DAC Report

### CODE REVIEW

**Comment by cleargif** [comment]
@colinbruce can you code review please..finally got the tests to pass using the longer `continue` procedure

File: app/views/external_users/claims/interim_fee/_fields.html.haml [hunk 1]
```diff
@@ -85,6 +85,6 @@
 
     %hr
 
-    .form-group
+    #disbursements.form-group
```
Comment (colinbruce) [inline app/views/external_users/claims/interim_fee/_fields.html.haml:88]: Should this be `interim_fees`?  

Or are disbursements sometimes known as different things?

File: features/page_objects/sections/disbursement_section.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@ class DisbursementSection < SitePrism::Section
   include SelectHelper
 
   element :select_container, "select.typeahead", visible: false
```
Comment (colinbruce) [inline features/page_objects/sections/disbursement_section.rb]: Can the old selector be deleted now?

**Review by colinbruce — COMMENTED** [review]
A couple of questions for you...

File: app/views/external_users/claims/interim_fee/_fields.html.haml (hunk 1 above)
Comment (cleargif) [reply app/views/external_users/claims/interim_fee/_fields.html.haml:88]: There is that strange combination form on interim fees.. This ID is used as a hook to scope the disbursement functionality

File: features/page_objects/sections/disbursement_section.rb (hunk 2 above)
Comment (cleargif) [reply features/page_objects/sections/disbursement_section.rb]: ahh.. let me check..

**Review by colinbruce — APPROVED** [review]

**Pull request merged**